inc/azure_c_util/rc_string.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef RC_STRING_H
+#define RC_STRING_H
+
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"
+#include "azure_c_util/thandle.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    typedef struct RC_STRING_TAG
+    {
+        const char* string;
+    } RC_STRING;
+
+    THANDLE_TYPE_DECLARE(RC_STRING);
```

Comment:
nit: seems like it shouldn't be indented if line 28 isn't.

Reply:
I indented all of them :-)

---
In reply to: [451153708](https://github.com/Azure/azure-c-util/pull/49#discussion_r451153708) [](ancestors = 451153708)